{"uuid": "0643829f-dc6b-42e5-b2ca-023048344695", "children": ["3e0c7988-51e3-4f86-953b-b778f5a242a0"], "befores": [{"name": "init_login_success", "status": "passed", "start": 1738051999093, "stop": 1738051999093}], "afters": [{"name": "init_login_success::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: disconnected: not connected to DevTools\n  (failed to check if window was closed: disconnected: not connected to DevTools)\n  (Session info: chrome=131.0.6778.267)\n  (Driver info: chromedriver=131.0.6778.87 (ce31cae9487357cfd3bd62984ed5250121a71a23-refs/branch-heads/6778@{#2287}),platform=Windows NT 10.0.22631 x86_64)\n\n", "trace": "  File \"C:\\Users\\T\\Desktop\\python\\pytest\\pythonProject\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\T\\Desktop\\python\\pytest\\pythonProject\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\T\\Desktop\\毕设\\NWU_Campus_UI_Auto_Test\\script\\test_login.py\", line 22, in init_login_success\n    IndexPage().logout()\n  File \"C:\\Users\\T\\Desktop\\毕设\\NWU_Campus_UI_Auto_Test\\page\\index_page.py\", line 53, in logout\n    raise e\n  File \"C:\\Users\\T\\Desktop\\毕设\\NWU_Campus_UI_Auto_Test\\page\\index_page.py\", line 48, in logout\n    driver.execute_script('window.localStorage.clear()')\n  File \"C:\\Users\\T\\Desktop\\python\\pytest\\pythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 465, in execute_script\n    return self.execute(Command.EXECUTE_SCRIPT, {\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\T\\Desktop\\python\\pytest\\pythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 238, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\T\\Desktop\\python\\pytest\\pythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 193, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1738052006246, "stop": 1738052006250}], "start": 1738051999093, "stop": 1738052006250}